torch_xla/csrc/tensor_methods.cpp

```diff
@@ -1183,8 +1183,10 @@ void eye_out(XLATensorPtr& out, int64_t lines, int64_t cols) {
 }
 
 void fill_(XLATensorPtr& input, const at::Scalar& value) {
+  SymIntElements sym_int_elements(input->GetIrValue(), input->shape());
```

Comment:
can you add a comment here to explain why we need `sym_int_elements` here? Something along the line of 
```
Fill_ is implemented by expanding the pass in value to the same shape of input and replace input Tensor's IR with the expanded value. In order to support input with dynamic shapes, we need to expand the value to its dynamic dimension hence we need to create a sym_int_elements here.
```